apps/backoffice/src/lib/be/institutions/business.ts

```diff
@@ -1,14 +1,8 @@
-import { AssistanceItem } from "@/generated/api/AssistanceItem";
-import { AssistanceResponse } from "@/generated/api/AssistanceResponse";
 import { Institution as BackofficeInstitution } from "@/generated/api/Institution";
 import { UserAuthorizedInstitution } from "@/generated/api/UserAuthorizedInstitution";
 import { UserAuthorizedInstitutions } from "@/generated/api/UserAuthorizedInstitutions";
 import { InstitutionResource } from "@/types/selfcare/InstitutionResource";
-import {
-  getInstitutionById,
-  getUserAuthorizedInstitutions,
-  sendSupportRequest
```

Comment:
Is the method `sendSupportRequest` removed also on selfcare-client.ts?

Reply:
de2731c9 _(I was forgetting it, thanks)_